core/templates/dev/head/components/entity-creation-services/exploration-creation.service.ts

```diff
@@ -88,47 +88,51 @@ angular.module('oppia').factory('ExplorationCreationService', [
               };
 
               $scope.cancel = function() {
-                $uibModalInstance.dismiss('cancel');
+                $uibModalInstance.close('cancel');
                 AlertsService.clearWarnings();
               };
             }
           ]
         }).result.then(function(result) {
-          var yamlFile = result.yamlFile;
+          if(result!='cancel') {
```

Comment:
Missing space before (, missing spaces around !=

Reply:
Done removed this Thansk!